src/index.ts

```diff
@@ -133,8 +145,12 @@ yargs.command({
 
             options = new mgmtApi.Options();
             options.token = token.access_token;
-            
-            let pushSync = new push(options, multibar);
+
+            let user = await auth.getUser(guid, token.access_token);
+            if(user){
+                console.log(colors.yellow('Pushing your instance...'));
+                let pushSync = new push(options, multibar);
+
             /*
         TODO: Inquirer for Content and Pages.
```

Comment:
Is this supposed to commented out as TODO still?

Reply:
Yes, for now, Inquirer is a TODO because we have to find out duplicate content and pages. The first version will be pushing the entire instance.